lib/rubygems/commands/install_command_extend.rb

```diff
@@ -13,10 +13,9 @@
 # limitations under the License.
 
 require 'rubygems/command_manager'
+require "rubygems/user_interaction"
 require 'rubygems/sigstore'
 
-Gem::CommandManager.instance.register_command :verify
-
 # gem install hooks
 i = Gem::CommandManager.instance[:install]
 i.add_option("--[no-]verify",
```

Comment:
This still needs to be updated to `--verify-signatures`. Will do this in a smaller PR.